boards/lilygo/ttgo_lora32/ttgo_lora32_esp32_appcpu.yaml

```diff
@@ -0,0 +1,27 @@
+identifier: ttgo_lora32/esp32/appcpu
+name: ttgo lora32 APPCPU
```

Comment:
I suggest `TTGO LoRa32 APPCPU`.

Reply:
Hello Sylvio,
thanks for your review. I applied the requested changes and pushed them.
I hope it can be merged into the 3.7 release